test/test_autograd.py

```diff
@@ -3482,6 +3482,13 @@ def test():
         test()
         self.assertEqual(dealloc[0], 1)
 
+    def test_inplace_view_leaf_errors(self):
+        # Issue #21875: Fail faster (when we try to modify the view vs. in backward())
+        x = torch.zeros(1, requires_grad=True)
+        y = x
```

Comment:
This is not creating a view :D 
Just assigning the python variable named `y` to point to the same thing as the python variable `x`.
So both `x` and `y` point to the same thing here.

If you want to create a view, you need to do something with the Tensor like: `y = x.view_as(x)`.

Reply:
Got it, thanks! :)